script/release.sh

```diff
@@ -7,7 +7,7 @@ WORKDIR=$(pwd)
 
 export DEBIAN_FRONTEND=noninteractive
 
-apt-get update
+apk update
```

Comment:
We should remove this whole line, it shouldn't be needed at all.

Reply:
that SGTM!